test-clients/src/main/java/com/hedera/services/bdd/suites/contract/records/LogsSuite.java

```diff
@@ -133,12 +138,14 @@ private HapiApiSpec log3Works() {
 														parsedToByteString(1),
 														parsedToByteString(2),
 														parsedToByteString(3))))
-										).gasUsed(23_638))));
+										).gasUsed(23_638))),
+						UtilVerbs.resetAppPropertiesTo("src/main/resource/bootstrap.properties"));
```

Comment:
`resetAppPropertiesTo` works by sending an Upload file transaction to the hedera node which overrides the current properties and then syncs them between the hedera nodes. We have added it to the test cases that override some property to make sure that we have the defaults for the next test execution. 
We could add it to some higher level component that is executed before each test but this would add an additional transaction for every test case which may not be required.